File: WEKEE.BE/Account.Domain/Shared/DataTransfer/PermisionDTO/PermissionFtReourceReadDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Account.Domain.Shared.DataTransfer.PermisionDTO
{
    public class PermissionFtReourceReadDto
    {
        public int Id { get; set; }
        public int ResourceId { get; set; }
        public string ResourceName { get; set; }
        public int PermissionId { get; set; }
        public string PermissionName { get; set; }
        public bool? IsActive { get; set; }
        public int CreateBy { get; set; }
        public string CreateName { get; set; }
        public DateTime CreatedOnUtc { get; set; }
        public DateTime UpdatedOnUtc { get; set; }
    }
}

File: WEKEE.FE/UI.HOME/src/scenes/Admin/APermission/dto/permissionFtReourceReadDto.ts
export interface PermissionFtReourceReadDto {
    id: number;
    resourceId: number;
    resourceName: string;
    permissionId: number;
    permissionName: string;
    isActive: boolean | null;
    createBy: number;
    createName: string;
    createdOnUtc: string;
    updatedOnUtc: string;
}